constellations/gemini-user/src/internship/internship.sass

```diff
@@ -11,6 +11,11 @@
         margin-bottom: 30px
         color: $colorTurquoise
 
+    & h4
+        margin-bottom: 20px
```

Comment:
When using hard-coded pixels, use the variables `spacing1x` found in `utils/constants.sass` instead. It keeps the spacing around the website more consistent.

Spacings like with margins or paddings should use `spacingxx`. Width and height of elements should use `propUnitxx`. Can you please replace all the values in this file with the appropriate variables?